packages/volto/src/components/manage/Contents/ContentsUploadModal.jsx

```diff
@@ -205,6 +212,32 @@ class ContentsUploadModal extends Component {
    * @returns {string} Markup for the component.
    */
   render() {
+    const dropzoneOptions = {};
+
+    const {
+      multiple = true,
+      minSize = null,
+      maxSize = null,
+      accept = null,
+      disabled = false,
+    } = this.props;
+
+    if (multiple) {
+      dropzoneOptions['multiple'] = multiple;
+    }
+    if (minSize) {
+      dropzoneOptions['minSize'] = minSize;
+    }
+    if (maxSize) {
+      dropzoneOptions['maxSize'] = maxSize;
+    }
+    if (accept) {
+      dropzoneOptions['accept'] = accept;
+    }
+    if (disabled) {
+      dropzoneOptions['disabled'] = disabled;
+    }
```

Comment:
```suggestion
   const dropzoneOptions = {
      multiple,
      minSize,
      maxSize,
      accept,
     disabled,
};
```

@erral you could simplify this code as such and you wouldn't need to have all of these if statements.
I assume that the drag and drop library would know to handle even the options that have null values

Reply:
Thanks @ichin-david surely it will. I am still a react novice for such structures and still go for if-like thingies. 😅 I will check if the library handles the null values correctly.

Reply:
I have checked this and indeed the plugin handless the null values correctly, so I have updated the PR. Thanks again for the review.